PORTAL/jobs/__init__.py

```diff
@@ -248,20 +258,26 @@ class JobsFS(_utils.FSTree):
     JOBFS = JobFS
 
     @classmethod
-    def from_user(cls, user, context='portal'):
-        return cls(f'/home/{user}/BENCH', context)
+    def from_user(cls, worker_name, user, context='portal'):
+        return cls(worker_name, f'~{user}/BENCH', context)
```

Comment:
With a harder separation between `JobsFS` and `WorkerJobsFS` (see #78), we can handle this a bit cleaner.

For instance, we can leave it like it was on `JobFS`, and on `WorkerJobsFS` we can adjust it depending on the worker OS.

Reply:
Yeah -- just looked at #78.  I agree that would help.  Let's get that landed, and then I can reframe this in terms of that.  It's bound to simplify things.

Reply:
> For instance, we can leave it like it was on JobFS, and on WorkerJobsFS we can adjust it depending on the worker OS.

A better separation between "paths for the portal" and "paths for the worker" totally makes sense. But I did learn yesterday as I was working on all this, that I think it's best to resolve paths for the worker on the worker.  There's a bunch of edge cases where the transformation is more than just `~` -> `$HOME`, which maybe won't matter for us in practice, but on Windows it definitely will.

Once all of the worker paths are "quarantined" in #78, I'd argue for a comment that says something like:

```
All paths on the worker should remain in "tildefied form", and be resolved later on the worker machine itself.
```

(Though maybe there's a clearer way to say it than "tildefied form").